<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WeatherStation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo/logo.qrc">
    <normaloff>:/rc/icon.png</normaloff>:/rc/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>671</width>
      <height>177</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>19</number>
     </property>
     <item>
      <widget class="QGroupBox" name="Group1">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="PressionIco">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="logo/logo.qrc">:/rc/pression.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="PressionLabel">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pression atmosph√©rique&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>Pression</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="Pression">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>value</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="VentIco">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="logo/logo.qrc">:/rc/vent.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="PressionLabel_2">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vitesse du vent&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>Vent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="Vent">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>value</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="RapporterIco">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="logo/logo.qrc">:/rc/rapporteur.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="RapporterLabel">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angle du vent&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>Angle Vent</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="Rapporter">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="toolTip">
               <string/>
              </property>
              <property name="whatsThis">
               <string/>
              </property>
              <property name="text">
               <string>value</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="leftMargin">
        <number>62</number>
       </property>
       <item>
        <widget class="QLabel" name="TimeIco">
         <property name="maximumSize">
          <size>
           <width>42</width>
           <height>39</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="logo/logo.qrc">:/rc/time.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Time">
         <property name="text">
          <string>temps</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="Group2">
       <property name="styleSheet">
        <string notr="true">/*border: 0</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="RefroidissementICO">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>41</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>41</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="logo/logo.qrc">:/rc/eolien.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="RefroidissementLabel">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Refroidissement √©olien.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Refroidissement</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Refroidissement">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>value</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="GPSIco">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>1</width>
              <height>1</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>21</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="logo/logo.qrc">:/rc/gps.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="GPSLabel">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nombre de satellites&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>GPS Satellites</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="GPS">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>value</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="DirectionICO">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="logo/logo.qrc">:/rc/aiguille.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="DirectionLabel">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Direction du port m√©t√©o&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Direction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Direction">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>value</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="LabelTemp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>701</width>
      <height>38</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Temp√©rature de l'air.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom-width: 1px;
border-bottom-style: solid;
border-radius: 0px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>temperature</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelError">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>551</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Veuillez aller dans Programme &gt; Param√®tres
Puis s√©lectionner le port COM valide
.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Invalide">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>null</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TempIco">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>31313</width>
      <height>1331313</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0)</string>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="logo/logo.qrc">:/rc/temperature.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgramme">
    <property name="title">
     <string>Programme</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionGraphique"/>
    <addaction name="actionConsole"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAutres">
    <property name="title">
     <string>Autres</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuProgramme"/>
   <addaction name="menuAutres"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="logo/logo.qrc">
     <normaloff>:/rc/Martz90-Circle-Settings.512.ico</normaloff>:/rc/Martz90-Circle-Settings.512.ico</iconset>
   </property>
   <property name="text">
    <string>Param√®tres</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="logo/logo.qrc">
     <normaloff>:/rc/close.png</normaloff>:/rc/close.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="icon">
    <iconset resource="logo/logo.qrc">
     <normaloff>:/rc/console.png</normaloff>:/rc/console.png</iconset>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionGraphique">
   <property name="icon">
    <iconset resource="logo/logo.qrc">
     <normaloff>:/rc/graphique.png</normaloff>:/rc/graphique.png</iconset>
   </property>
   <property name="text">
    <string>Graphique</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="logo/logo.qrc"/>
 </resources>
 <connections/>
</ui>
